controllers/UsersController.js

```diff
@@ -83,4 +85,63 @@ export default class UsersController {
       }
     }).catch(next);
   }
+
+  /** @param {Object} req - Request object
+  * @param {Object} res - Response object
+  * @returns {Object} success or failure response
+  */
+  static sendEmail(req, res) {
+    let transport;
+    const mailOptions = {
+      from: 'sinmi.mtracker@gmail.com',
```

Comment:
You can fetch this is an environment variable.

Reply:
Alright, thanks, I will implement the change